pkg/receive/handler.go

```diff
@@ -374,16 +380,21 @@ func (h *Handler) fanoutForward(ctx context.Context, tenant string, replicas map
 		if !replicas[endpoint].replicated && h.options.ReplicationFactor > 1 {
 			go func(endpoint string) {
 				defer wg.Done()
+
 				var err error
 				tracing.DoInSpan(ctx, "receive_replicate", func(ctx context.Context) {
 					err = h.replicate(ctx, tenant, wreqs[endpoint])
 				})
 				if err != nil {
+					h.replications.WithLabelValues("error").Inc()
 					ec <- errors.Wrapf(err, "replicate write request, endpoint %v", endpoint)
 					return
 				}
+
+				h.replications.WithLabelValues("success").Inc()
```

Comment:
I would suggest adding `success` and `error` to constant vars so we can reduce chance of typo increasing cardinality and creating massive confusion ;p